### PR TITLE
Feature/redux segment

### PR DESCRIPTION
**Pull request by marina-acosta**
Track user actions: Integration with segment.

### CODE REVIEW

File: client/components/trackedForm/TrackedForm.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { submitTrackedForm } from './tracked.form.action';
+//import styles from '.TrackedForm.scss';
```
Comment (carloluis) [inline client/components/trackedForm/TrackedForm.jsx]: * remove comments
* `TrackedForm.scss` is empty

File: client/components/trackedForm/tracked.form.reducer.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { FORM_SUBMIT } from './tracked.form.action';
+
+const DEFAULT_STATE = {
+	tax: 15,
+	fee: 0.5
+}
```
Comment (carloluis) [inline client/components/trackedForm/tracked.form.reducer.js]: `;` missing

File: client/config/webpack.config.common.js [hunk 3]
```diff
@@ -59,8 +60,10 @@ const common = {
 				preserveLineBreaks: true,
 				useShortDoctype: true,
 				html5: true
-			}
+			},
+			scripts: ['segment/segment.js']
 		}),
+		new CopyWebpackPlugin([{ from: PATHS.app + '/segment/segment.js', to: PATHS.build + '/segment/segment.js'}]),
```
Comment (carloluis) [inline client/config/webpack.config.common.js]: can we minify `segment.js` script?

File: client/components/trackedForm/TrackedForm.jsx (hunk 1 above)
Comment (matiasdelgado) [inline client/components/trackedForm/TrackedForm.jsx]: Also, the file name should be `tracked-form.action` (with a dash rather than a dot) :)

**Pull request merged**